src/org/omegat/core/machinetranslators/ApertiumTranslate.java

```diff
@@ -83,18 +94,23 @@ private String apertiumCode(Language language) {
 
     @Override
     protected String translate(Language sLang, Language tLang, String text) throws Exception {
-        String prev = getFromCache(sLang, tLang, text);
-        if (prev != null) {
-            return prev;
-        }
-
```

Comment:
Did you mean to delete the caching mechanism? That doesn't seem desirable to me.

Reply:
Sorry, I removed it for testing and forgot to add it back. It is now in code again.